localservices/cloudapi/service.go

```diff
@@ -40,6 +40,12 @@ type CloudAPI struct {
 	fabricVLANs   map[int16]*fabricVLAN
 }
 
+type machine struct {
+	cloudapi.Machine
+	NICs        map[string]*cloudapi.NIC `json:"-"`
+	NetworkNICs map[string]string        `json:"-"`
+}
+
```

Comment:
Why the new machine struct here? Is this better than modifying the existing cloudapi.machine?